interfaces/builtin/dsp.go

```diff
@@ -49,10 +49,16 @@ const ambarellaDspConnectedPlugApparmor = `
 # processor
 /dev/ucode rw,
 
-# The iav device node is the device node exposed for the specific IAV linux
-# device driver used on the ambarella device
+# The iav device node is the device node exposed for the specific IAV driver
+# that includes a set of Linux device drivers that encapsulate the lower-level
+# complexities of the CV2x / S6Lm DSP core functionalities
 /dev/iav rw,
 
+# The cavalry device node is the driver for the CV2x vector processor (VP)
```

Comment:
I'm not sure what "the driver" refers to here?   it means the control device?  is just the node "for the CV2x VP" ?